[OSDOCS-17003] IBM Z callout replacement: replace AsciiDoc callouts with definition … (#116250)
openshift/openshift-docs: modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc

   --memory <memory_mb> \
   --vcpus <vcpus> \
   --disk <disk> \
   --launchSecurity type="s390-pv" \
   --import \
   --network network=<virt_network_parm>,mac=<mac_address> \
   --disk path=<ign_file>,format=raw,readonly=on,serial=ignition,startup_policy=optional
----
+

> The `--launchSecurity type="s390-pv"` parameter is required only if {ibm-name} Secure Execution is enabled. If {ibm-name} Secure Execution is enabled, also replace `serial=ignition` with `serial=ignition_crypted` in the `--disk` parameter.

Reviewer: This is fine but as another tip, if you wanted to do a more minimal change for future callouts like this, this seems written in a way where you could replace the callouts with list items, like this:

```suggestion
* If {ibm-name} Secure Execution is enabled, add the `launchSecurity type="s390-pv"` parameter.
* If {ibm-name} Secure Execution is enabled, replace `serial=ignition` with `serial=ignition_crypted`.
```

You don't have to change what you have, only wanted to let you know about another option
